### DISCUSSION TITLE
Woocommerce apps camera needs to read GTIN/EAN barcode

### DISCUSSION THREAD

**Discussion by ojosejefferson** [woocommerce/woocommerce#51131] [General]
I am testing to apply woocommerce 100% in my physical store using woocommerce applications, however when using the application's camera the camera only reads SKU I would like it to also read GTIN/EAN barcodes it would help a lot to control the stock because we do this through the barcode and not just the SKU. Could there be a native option or in the settings to read SKU and/or EAN to locate the products on the wocommerce website?

**Comment by moon0326** [comment]
Hi @ojosejefferson 

Could you please expand on that a little further? Which plugins (applications?) are you using to scan your SKUs/barcodes?

**Reply by ojosejefferson** [reply]
![20240906_154121.jpg](https://github.com/user-attachments/assets/109536dd-5dfa-4ce8-a1fe-baae09fecb0b)

Hello, I'm using the woocommerce application itself, I would like to read the Gtin/EAN in addition to the sku, even with filling it in on the website it does not locate the product.

**Reply by moon0326** [reply]
@ojosejefferson My apologies, I didn’t realize you were referring to the iOS WooCommerce app.

For the supported barcode types, please refer to this link: https://woocommerce.com/document/barcode-and-qr-code-scanner/

I’ve tested a few different GTIN and EAN barcodes, and they worked for me.

Could you share a barcode that isn’t working so I can test it?

**Reply by ojosejefferson** [reply]
My site exemple:  

https://maksantos.com.br/jogo-de-brocas-para-concreto-com-5-pecas-d-03894-makita/ 

GTIN
088381174268

My config product store 

![20240906_164901.jpg](https://github.com/user-attachments/assets/ab1715aa-6572-4677-b225-9940cc109270)


It would be cool if the WooCommerce app reader searched by EAN/GTiN as well and not just by SKU

**Reply by moon0326** [reply]
Thank you for clarifying. I'll create a new issue to add support for GTIN, UPC, EAN, or ISBN search functionality.

Update: https://github.com/woocommerce/woocommerce-ios/issues/13881 has been created.

**Reply by ojosejefferson** [reply]
Thank you
